anti_alias_freq: 20.0

#Dynamical parameters
max_vel: 3.0
max_eff: 5.0
stiff: 1.0

#Homing
has_home: 0
upper_bounds: [0.0, 0.0]
lower_bounds: [0.0, 0.0]
origins: [0.0, 0.0]
